### ISSUE TITLE
multiple carousels on the same page set base options for the next ones

### ISSUE THREAD

**Issue by rainer-friederich** [WondrousLLC/atomic-slider#5]
to see here: 
https://www.dev-54ta5gq-densryw4wsozu.eu.platform.sh/de/referenzen/k204-wiggertalstrasse-neubau-betonkreisel-bernstrasse

The first carousel sets items: 4 and margin: 10

The next carousel inherits this and i need to reset the values.

A carousel should never inherit options

**Comment by anton-staroverov** [comment]
@yobottehg if you'll share the `rapp` code - I can help to investigate! `atomic-carousel` itself seems to work correctly

**Comment by tassilogroeper** [comment]
@yobottehg  can you be more specific about the problem? With the link provided, I can not reproduce margins beeing inherited also Item count is correct on the second carousel?!

**Comment by rainer-friederich** [comment]
@pheadeaux yes because i provide overwrite values `data-carousel="{"loop":true,"items":1,"margin":0}"` it works. if you leave them away the carousel will have 4 items.
